### ISSUE TITLE
Support to change Picker popup background color

### ISSUE THREAD

**Issue by Kerubananthan3761** [dotnet/maui#15316]
### Description

In MAUI Picker there is no support to customize picker popup background 

### Public API Changes

PopupBackground in Picker

### Intended Use-Case

In .Net MAUI DataForm Control, while providing support for theme, there is support to change the color of Picker background and text color which are contrast to dark and light theme. We are not able to change Picker popup background color.

**Comment by rachelkang** [comment]
Hi, @Kerubananthan3761 - could you clarify which platform you're particularly seeking this on, and which popup portion of the picker you are referring to? Thanks!

**Comment by Kerubananthan3761** [comment]
Hi @rachelkang , I am currently working on Windows platform, but I need to consider for other platform also. Refer the below image, which has picker items. Need to customize the background of the items.
![image](https://github.com/dotnet/maui/assets/104969840/86144359-f6d5-43ef-86a0-9a66941eb84c)

**Comment by jeremy-visionaid** [comment]
@rachelkang More that just the requested feature, unfortunately the foreground and background for the popup does not change when setting `Application.Current.UserAppTheme` (tested on 8.0.0-preview.7.8842)

@Kerubananthan3761 At least a workaround is available by adding resource for the underlying control:
https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.comboboxitem

**Comment by malsabi** [comment]
I'm having the same issue on Picker Control the BackgroundColor of the Menu Dropdown can't be changed specially when setting Application.Current.UserAppTheme to Light, the BackgroundColor of the Menu Dropdown remains black and vice versa.

**Comment by ChrisK91** [comment]
I'm running into the same issue. @jeremy-visionaid, were you able to work around the issue where the color is not updated on theme changes?

**Comment by jeremy-visionaid** [comment]
@ChrisK91 Yeah, you can add brushes for `ComboBoxItemBackground` etc, to a `Microsoft.UI.Xaml.ResourceDictionary`. The distribution of the upstream styles has changed, but you should still be able to find all the references you need from the links above.

**Comment by AndreasReitberger** [comment]
+1
Any news on this? 
The picker uses the device theme of the computer (Windows).

<img width="1086" height="343" alt="Image" src="https://github.com/user-attachments/assets/56d62f57-ccb6-4f67-a53d-7df6554578e7" />

**Comment by malsabi** [comment]
I think this PR https://github.com/dotnet/maui/pull/31714 will solve this issue.

**Comment by AndreasReitberger** [comment]
> I think this PR [#31714](https://github.com/dotnet/maui/pull/31714) will solve this issue.

Awesome, maybe also this one?
https://github.com/dotnet/maui/issues/31819
